src/config/routes/2.1.json

```diff
@@ -144,6 +144,13 @@
     },
 
     {
+        "path": "/users(/[^/]+)*/?$",
+        "controller": "UsersController",
+        "action": "deleteAction",
```

Comment:
I'm not precious about it either way - most of the code was a direct copy from another deleteAction if I'm honest - will change the method name and commit.